flake8_routable.py

```diff
@@ -416,6 +419,51 @@ def disallow_feature_flag_creation(self) -> None:
             reported.add(line_token.start[0])
             self.errors.append((*line_token.start, ROU111))
 
+    def tasks_without_args_and_kwargs(self) -> None:
+        """Don't allow tasks without args or kwargs."""
```

Comment:
This also checks for `priority`, let's reflect that in the method name and docstring.

How about:
```suggestion
    def task_args_kwargs_and_priority(self) -> None:
        """Don't allow tasks without args or kwargs and ensure priority is not allowed as a task kwarg."""
```

Reply:
updated here: dda5de57f9942c57c2db8196e9a480a560ae66eb